fv3_cap.F90

```diff
@@ -0,0 +1,1691 @@
+!--------------- FV3GFS solo model -----------------
+!
+!*** The FV3 atmosphere grid component nuopc cap
+!
+! Author:  Jun Wang@noaa.gov
+!
+! revision history
+! 11 Oct 2016: J. Wang          Initial code
+! 18 Apr 2017: J. Wang          set up fcst grid component and write grid components
+! 24 Jul 2017: J. Wang          initialization and time stepping changes for coupling
+! 02 Nov 2017: J. Wang          Use Gerhard's transferable RouteHandle
+!
+
+module fv3gfs_cap_mod
+
+  use ESMF
+  use NUOPC
+  use NUOPC_Model,            only: model_routine_SS        => SetServices,       &
+                                    model_routine_Run       => routine_Run,       &
+                                    model_label_Advance     => label_Advance,     &
+                                    model_label_CheckImport => label_CheckImport, &
+                                    model_label_Finalize    => label_Finalize,    &
+                                    NUOPC_ModelGet
+!
+  use module_fv3_config,      only: quilting, restart_interval,              &
+                                    nfhout, nfhout_hf, nsout, dt_atmos,      &
+                                    nfhmax, nfhmax_hf,output_hfmax,          &
+                                    output_interval,output_interval_hf,      &
+                                    alarm_output_hf, alarm_output,           &
+                                    calendar, calendar_type, cpl,            &
+                                    force_date_from_configure,               &
+                                    cplprint_flag,output_1st_tstep_rst,      &
+                                    first_kdt,num_restart_interval
+
+  use module_fv3_io_def,      only: num_pes_fcst,write_groups,app_domain,    &
```

Comment:
Are these IO parameters needed at this level?

Reply:
In the code further down, these fields are used to set up ESMF alarm.